iphone/cli/hooks/run.js

```diff
@@ -72,6 +73,14 @@ exports.init = function (logger, config, cli) {
 					lastLogger = m[2].toLowerCase();
 					line = m[4].trim();
 				}
+
+				// ignore logs from cli ignoreLog
+				for (let i = 0, len = ignoreLog.length; i < len; ++i) {
+					if (line.includes(ignoreLog[i])) {
+						return;
+					}
+				}
```

Comment:
A possibly cleaner approach:
```js
if (ignoreLog.some(ignoreItem => line.includes(ignoreItem))) {
    return;
}
```

Reply:
I like that, let's use that for the existing Android one too :+1: